eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1545,9 +1545,21 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
                 ()
             }
           }
+
           // we will re-enable the channel after some delay to prevent flappy updates in case the connection is unstable
           setTimer(Reconnected.toString, BroadcastChannelUpdate(Reconnected), 10 seconds, repeat = false)
 
+          // We usually handle feerate updates once per block (~10 minutes), but when our remote is a mobile wallet that
+          // only briefly connects and then disconnects, we may never have the opportunity to send our `update_fee`, so
+          // we send it (if needed) when reconnected.
+          if (d.commitments.localParams.isFunder) {
+            val currentFeeratePerKw = d.commitments.localCommit.spec.feeratePerKw
```

Comment:
We should look at the remote commit, because we may already have a pending `update_fee`, and it will first appear in the remote commitment. Btw this means that `handleCurrentFeerate` is bugged too.

```suggestion
            val currentFeeratePerKw = d.commitments.remoteCommit.spec.feeratePerKw
```

Reply:
Actually, we really care about our commitment, so shouldn't we also take the local commitment into consideration? Damn...

Reply:
I think we really want it to be applied to our local commit. I don't think it matters much if we end up sending a duplicate `update_fee`, the spec doesn't prevent that so better be safe here and look at our commit?

Reply:
I was afraid of creating a loop but we can only having one pending sig so I guess we're good.